src/audio/smart_amp/smart_amp_maxim_dsm.c

```diff
@@ -18,11 +18,48 @@
 #include <stdint.h>
 #include <stdlib.h>
 #include <sof/audio/smart_amp/smart_amp.h>
+#include "dsm_api_public.h"
 
 static int maxim_dsm_init(struct smart_amp_mod_struct_t *hspk, struct comp_dev *dev)
 {
-	/* Maxim Dynamic Speaker Manangement initialization */
+	int *circularbuffersize = hspk->circularbuffersize;
+	int *delayedsamples = hspk->delayedsamples;
+	struct dsm_api_init_ext_t initparam;
+	enum DSM_API_MESSAGE retcode;
+
+	comp_dbg(dev, BUILD_TIME);
```

Comment:
Should this be removed ? It will make the build non reproducible.

Reply:
Agreed. It should be removed.